src/main/java/com/dev/theater/util/HibernateUtil.java

```diff
@@ -0,0 +1,24 @@
+package com.dev.theater.util;
+
+import com.dev.theater.exception.CrudException;
+import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
+
+public class HibernateUtil {
+    private static final SessionFactory sessionFactory = initSessionFactory();
+
+    private HibernateUtil(){
+    }
+
+    public static SessionFactory getSessionFactory() {
+        return sessionFactory;
+    }
+
+    private static SessionFactory initSessionFactory() {
+        try {
+            return new Configuration().configure().buildSessionFactory();
+        } catch (Exception e) {
+            throw new CrudException("Error creating Session Factory", e);
```

Comment:
it isn't crud problem, let's throw runtime exception here

Reply:
done